store/mockstore/mockcopr/executor_test.go

```diff
@@ -28,74 +27,75 @@ import (
 	"github.com/pingcap/tidb/store/mockstore/mockcopr"
 	"github.com/pingcap/tidb/store/mockstore/mockstorage"
 	"github.com/pingcap/tidb/tablecodec"
-	"github.com/pingcap/tidb/util/testkit"
+	"github.com/pingcap/tidb/testkit"
+	"github.com/stretchr/testify/require"
 	"github.com/tikv/client-go/v2/oracle"
 	"github.com/tikv/client-go/v2/testutils"
 	"github.com/tikv/client-go/v2/tikv"
 )
 
-func TestT(t *testing.T) {
-	CustomVerboseFlag = true
-	TestingT(t)
-}
-
-var _ = Suite(&testExecutorSuite{})
-
 type testExecutorSuite struct {
 	cluster   *testutils.MockCluster
 	store     kv.Storage
 	mvccStore testutils.MVCCStore
 	dom       *domain.Domain
 }
 
-func (s *testExecutorSuite) SetUpSuite(c *C) {
+func SetUpSuite(t *testing.T) (s *testExecutorSuite, clean func()) {
```

Comment:
I don't think `SetUpSuite` a proper name. Let me try to push a follow-up for this PR.